microsoft-edge/devtools-guide-chromium/javascript/index.md

```diff
@@ -151,48 +155,52 @@ Line-of-code breakpoints are the most common type of breakpoint.  When you get t
+The values of `addend1`, `addend2`, and `sum` look suspicious.  These values are wrapped in quotes, which means each value is a string. This is a good clue to the cause of the bug.  The next step is to gather more information about these variable values.  DevTools provides many tools for examining variable values.
 
-### Method 1: The Scope pane
+#### Method 1: The Scope pane
 
 If you pause on a line of code, the **Scope** pane displays the local and global variables that are currently defined, along with the value of each variable.  It also displays closure variables, as applicable.  Double-click a variable value to edit it.  If you don't pause on a line of code, the **Scope** pane is empty.
 
 ![The Scope pane.](../media/javascript-sources-breakpoint-paused-scope.msft.png)
 
-### Method 2: Watch Expressions
+#### Method 2: Watch expressions
 
-The **Watch** pane allows you to monitor the values of variables (such as `sum`) or expressions (such as `typeof sum`).  You can store any valid JavaScript expression in a Watch Expression.
+The **Watch** pane allows you to monitor the values of variables (such as `sum`) or expressions (such as `typeof sum`).  You can store any valid JavaScript expression in a watch expression.
 
 1. Select the **Watch** tab.
 
 1. Click **Add watch expression** (![Add watch expression.](../media/add-expression-icon.msft.png)).
 
 1. Type `typeof sum`.
 
-1. Press `Enter`.  DevTools displays `typeof sum: "string"`.  The value to the right of the colon is the result of your Watch Expression.
+1. Press `Enter`.  DevTools displays `typeof sum: "string"`.  The value to the right of the colon is the result of your watch expression.
 
 > [!NOTE]
-> In the following figure, the `typeof sum` Watch Expression is displayed in the **Watch** pane.  If your DevTools window is wide, the **Watch** pane is displayed within the **Debugger** pane, which then appears on the right.
+> In the following screenshot, the watch expression `typeof sum` is displayed in the **Watch** pane.  If your DevTools window is wide, the **Watch** pane is displayed within the **Debugger** pane, which then appears on the right.
 
 ![The Watch pane.](../media/javascript-sources-breakpoint-paused-watch.msft.png)
 
 As suspected, `sum` is being evaluated as a string, when it should be a number.  You now confirmed value type is the cause of the bug.
 
-### Method 3: The Console
+#### Method 3: The Console
 
 The **Console** allows you to view `console.log()` output.  You can also use the **Console** to evaluate arbitrary JavaScript statements while the debugger is paused at a code statement.  For debugging, you can use the **Console** to test potential fixes for bugs.
 
-1. If the **Console** tool is closed, press `Esc` to open it.  The **Console** tool opens in the lower pane of the DevTools window.
+1. If the **Console** tool is closed, press `Esc` to open it.  The **Console** tool opens in the top pane of the DevTools window.
```

Comment:
```suggestion
1. If the **Console** tool is closed, press `Esc` to open it.  The **Console** tool opens in the lower pane of the DevTools window.
```
I tested Esc, always opens in lower pane on my machine if I open it by pressing Esc

Reply:
done